### DISCUSSION TITLE
国家天天强调要有监管账户，政府监管的资金到哪去了？

### DISCUSSION THREAD

**Discussion by Jackwew** [WeNeedHome/SummaryOfLoanSuspension#290] [Ideas]
各大房企频繁暴雷，监管账户空空如也

**Comment by ryzencool** [comment]
我自己监管我自己

**Comment by Shadow-Xin** [comment]
要是钱真的都进监管账户了，那国家还用天天强调吗？

**Comment by ljw-bigtail** [comment]
银行拒不执行国家的监管账户相关政策，挑拨人民内部矛盾了属实是

**Comment by xujiyou** [comment]
被分了呗，银行管事的、各级政府、开发商，养了这么多吸血鬼

**Comment by jbl19860422** [comment]
回答就是：你提的这个问题很好，我们将密切关注事态发展，永远执行为人民服务的政策。（然后第二天你就不见了）

**Comment by yourselfo** [comment]
https://user-images.githubusercontent.com/46863132/178928525-37b4e416-48c6-4147-84af-d2f4b66bd5a1.mp4

**Comment by 506124204** [comment]
反正马云敲醒大家，反而被骂。我看Z国有95%都是sa的可怜。
马云说了真话，你们不信。监Guan形同虚设。
房子如葱你们也不信。

**Reply by liuwenwu520** [reply]
马云也是想割韭菜，只是不是和银行一起的而已
